### PR TITLE
Update caption & image

### LINKED ISSUE #3: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @ericalloyd
- [ ] @dumpsterfire

### PR DESCRIPTION
**Pull request by dumpsterfire**
Updated the caption and the image to be more festive 🦃 💯

Closes #3

### CODE REVIEW

File: _posts/0000-01-03-dumpsterfire.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Happy Thanksgiving! 
 
-![droidtocat](https://octodex.github.com/images/droidtocat.png)
+![gobble-o-tron](https://octodex.github.com/images/gobbleotron.gif)
```
Comment (beardofedu) [inline _posts/0000-01-03-dumpsterfire.md:8]: This was a great change, very inline with the holidays 👍

File: _posts/0000-01-03-dumpsterfire.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Happy Thanksgiving!
```
Comment (beardofedu) [inline _posts/0000-01-03-dumpsterfire.md]: Although this is inline with the image, maybe we could try to be funnier? 

@ericalloyd to get your opinion on this

File: _posts/0000-01-03-dumpsterfire.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-03-dumpsterfire.md]:
```suggestion
title: "Welcome to dumpsterfire's slide!"
```

**Review by beardofedu — CHANGES_REQUESTED** [review]
Make sure to implement my suggested change before this is good to 🚢

File: _posts/0000-01-03-dumpsterfire.md (hunk 1 above)
Comment (dumpsterfire) [reply _posts/0000-01-03-dumpsterfire.md:8]: Thanks @beardofedu , your feedback is always so valuable!

File: _posts/0000-01-03-dumpsterfire.md (hunk 2 above)
Comment (dumpsterfire) [reply _posts/0000-01-03-dumpsterfire.md]: Yea, I'll definitely fix that, good idea! 💡

**Review by beardofedu — APPROVED** [review]
Thanks for adding all my changes 👍

**Pull request merged**